src/app/[locale]/_hooks/useLogin.tsx

```diff
@@ -27,8 +27,8 @@ const useLogin = (): LoginData => {
   const logOut = () => {
     setIsLoggedIn(false);
     setUserLogged(undefined);
-    cookieTokenOps.delete();
-    cookieUserOps.delete();
+    storageTokenOps.delete();
+    storageUserOps.delete();
```

Comment:
Grouping this two functions in a more explicit single one would be more expressive? What do you think? e.g. `clearSessionData()`

Reply:
Updated with sessionStorage.clear();